evo_researcher/autonolas/research.py

```diff
@@ -947,11 +947,6 @@ def extract_and_sort_sentences(
     doc_question = nlp(event_question)
     event_date = extract_event_date(doc_question)
 
-    if event_date is None:
```

Comment:
Removed the print because it's not used anyway, and most of the time, it's none.